docs/installing-solidity.rst

```diff
@@ -319,9 +319,9 @@ The following are dependencies for all builds of Solidity:
 +-----------------------------------+-------------------------------------------------------+
 | Software                          | Notes                                                 |
 +===================================+=======================================================+
-| `CMake`_ (version 3.13+)          | Cross-platform build file generator.                  |
+| `CMake`_ (version 3.21.3+)        | Cross-platform build file generator.                  |
```

Comment:
I actually meant
```suggestion
| `CMake`_ (version 3.21.3+ on Windows, 3.13+ otherwise)        | Cross-platform build file generator.                  |
```
:-). But yeah, we're starting to get lost in the details here... still fine either way...

Reply:
Honestly, that's a good point - so I'll make the change. Be on standby for another approval :)